src/plotman/manager.py

```diff
@@ -112,7 +160,8 @@ def maybe_start_new_plot(dir_cfg, sched_cfg, plotting_cfg):
             logfile = os.path.join(dir_cfg['log'],
                     datetime.now().strftime('%Y-%m-%d-%H:%M:%S.log'))
 
-            plot_args = ['chia', 'plots', 'create',
+            plot_args = [CHIA_EXECUTABLE.command, 'plots', 'create',
+                    '--override-k',
```

Comment:
Should `--override-k` be here?

Reply:
Heh, nope...  that was for testing.  Thanks.  (side note, we still need to add that to the config.  but that comes after #107)

```suggestion
```